main.go

```diff
@@ -52,36 +54,54 @@ func executeCommand(c *cli.Context) error {
 	interval := c.Uint("interval")
 
 	ctx := context.Background()
-	client, err := pubsub.NewClient(ctx, proj)
+
+	// https://github.com/google/google-api-go-client#application-default-credentials-example
+	client, err := google.DefaultClient(ctx, pubsub.PubsubScope)
 	if err != nil {
-		fmt.Println("Failed to get new pubsub client for ", proj, " cause of ", err)
-		os.Exit(1)
+		fmt.Printf("Failed to google.DefaultClient with scope %v cause of %v\n", pubsub.PubsubScope, err)
+		return err
+	}
+
+	// Creates a pubsubService
+	pubsubService, err := pubsub.New(client)
+	if err != nil {
+		fmt.Printf("Failed to Failed to create pubsub.Service with client %v cause of %v\n", client, err)
```

Comment:
s/Failed to Failed to create/Failed to create/

Reply:
@minimum2scp Thanks! I fixed it.